Simplified/NYPLTenPrintCoverView+NYPLImageAdditions.m

```diff
@@ -16,10 +16,22 @@ + (void)initialize {
   authorHeight = 15;
 }
 
-+ (UIImage *)imageForBook:(NYPLBook *const)book
++ (UIImage *)detailImageForBook:(NYPLBook *const)book
 {
-  CGFloat const width = 80;
-  CGFloat const height = 120;
+  return [NYPLTenPrintCoverView imageForBook:book withSize:CGSizeMake(510,680)];
+}
+
++ (UIImage *)thumbnailImageForBook:(NYPLBook *const)book
+{
+  return [NYPLTenPrintCoverView imageForBook:book withSize:CGSizeMake(80,120)];
+}
+
++ (UIImage *)imageForBook:(NYPLBook *const)book withSize:(CGSize)size
+{
+  CGFloat const width = size.width;
+  CGFloat const height = size.height;
+
+  Float32 scale = (width <= 80) ? 0.4 : 1.5;
```

Comment:
This overall change is in the right direction. And now with scale needing two values, it's good practice to take interdependencies away. What I mean is that when someone looks here they would not know why 0.4 and 1.5 would be chosen, so we're going to do the same thing we did with "size" and add one more parameter to the signature:

`+ (UIImage *)imageForBook:(NYPLBook *const)book withSize:(CGSize)size textScaleFactor:(CGFloat)scale`

And move the hard-coded values to the calls for "thumbnail" and "detailView"